app/controllers/home.ts

```diff
@@ -28,11 +28,26 @@ export default class HomeController extends Controller {
   ) {
     this.selectedMunicipalities = selectedMunicipalities;
   }
+  /** Handles keyword search on homepage */
+  @tracked value = '';
```

Comment:
**Nitpick:** Using a more meaningful name would improve readability and avoid confusion if other input text is added in the future 🙃